src/main/java/net/wouterdanes/docker/maven/StopContainerMojo.java

```diff
@@ -34,24 +34,8 @@
 
     @Override
     public void doExecute() throws MojoExecutionException, MojoFailureException {
-        for (StartedContainerInfo container : getStartedContainers()) {
-            String containerId = container.getContainerInfo().getId();
-            getLog().info(String.format("Stopping container '%s'..", containerId));
-            try {
-                getDockerProvider().stopContainer(containerId);
-            } catch (DockerException e) {
-                getLog().error("Failed to stop container", e);
-            }
-        }
-        for (StartedContainerInfo container : getStartedContainers()) {
-            String containerId = container.getContainerInfo().getId();
-            getLog().info(String.format("Deleting container '%s'..", containerId));
-            try {
-                getDockerProvider().deleteContainer(containerId);
-            } catch (DockerException e) {
-                getLog().error("Failed to delete container", e);
-            }
-        }
+        cleanUpStartedContainers();
+
         for (BuiltImageInfo image : getBuiltImages()) {
```

Comment:
There's a case to be made that the force cleanup should apply to built images too; I excluded it because a) it's more conservative and b) `forceCleanup` is a setting on `start-containers`, not `build-images`.

(From a design perspective it'd make more sense to clean built images in a separate mojo and require users to bind it explicitly, which would make this subsequent design choice the obvious right behaviour, but it seems too late for such a radical change in behaviour.)